database/migrations/2019_12_03_105400_create_action_stats_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateActionStatsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('action_stats', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('action_id')->index()->comment('The action id the stat is for.');
+            $table->foreign('action_id')->references('id')->on('actions');
+            $table->string('school_id', 255)->index()->comment('The school id the stat is for.');
+            $table->unsignedInteger('accepted_quantity')->comment('The sum of accepted post quantity for the school and action.');
+            $table->timestamps();
+            $table->softDeletes();
```

Comment:
Does this need to be soft-deletable? I'm having a hard time thinking of how we'd use this.